### PR TITLE
added basic test including some fixes

### PR DESCRIPTION
**Pull request by cameronbraid**

### CODE REVIEW

File: packages/cubejs-clickhouse-driver/package.json [hunk 1]
```diff
@@ -13,7 +13,11 @@
   "dependencies": {
     "@apla/clickhouse": "^1.5.5",
     "@cubejs-backend/query-orchestrator": "^0.0.21",
-    "generic-pool": "^3.6.0"
+    "generic-pool": "^3.6.0",
+    "mocha": "^5.2.0"
```
Comment (inikolaev) [inline packages/cubejs-clickhouse-driver/package.json]: Should `mocha` be inside of `devDependencies` instead?

**Comment by cameronbraid** [comment]
yep

File: packages/cubejs-clickhouse-driver/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+version: '2'
```
Comment (inikolaev) [inline packages/cubejs-clickhouse-driver/docker-compose.yml]: Let's set version to `3`, other than that the format of the file should be the same in our case.

**Comment by cameronbraid** [comment]
Sure

**Comment by inikolaev** [comment]
@cameronbraid Maybe we should actually try using https://github.com/testcontainers/testcontainers-node instead of Docker Compose for tests. The benefit is that it can be set up from the test itself and no need to run `docker-compose` manually. I use it for Java projects and it's very handy. This can be done as a follow up PR.

**Pull request merged**